backend/src/routes/BankingRoutes.ts

```diff
@@ -15,7 +17,7 @@ const PLAID_CLIENT_ID = process.env.PLAID_CLIENT_ID
 const PLAID_SECRET = process.env.PLAID_SECRET
 const PLAID_ENV = 'sandbox'
 
-let ACCESS_TOKEN = null
+let ACCESS_TOKEN:string = '';
```

Comment:
leave ALL_CAPS variables for static global variables. In fact, there's no need this needs to be a global variable, right?